README.rst

```diff
@@ -496,13 +496,13 @@ pytest-bdd feature file format also supports example tables in different way:
             When I eat <eat> cucumbers
             Then I should have <left> cucumbers
 
-            Examples: Vertical
+            Examples Transposed:
```

Comment:
Is it possible to separate tags from transposed examples into separate PRs? It is very hard to follow what's what

Reply:
#475